its/pom.xml

```diff
@@ -17,7 +17,7 @@
     <dependency>
       <groupId>org.sonarsource.orchestrator</groupId>
       <artifactId>sonar-orchestrator</artifactId>
-      <version>3.26.0.2111</version>
+      <version>3.33.0.2677</version>
```

Comment:
This version is not released

Reply:
but needed for the tests to pass, the same goes for the maven repo

Reply:
It is fine for testing.
To get this PR to master, a new orchestrator version needs to be released.